tes-adapter/scripts/run_get_tes-adapter_jar.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+workdir=$(pwd);
+
+if [ -n "$2" ]; then
+    gitRepoUrl="-b $1 $2"
+else
+    gitRepoUrl=$1
+fi
+
+mkdir tmp;
+cd tmp;
+tmpdir=$(pwd);
+
+#Step 1
+echo "Cloning git-repo from $gitRepoUrl"
+git clone $gitRepoUrl $tmpdir || { echo "Cloning failed!" >&2; exit 1; };
+
+#Step 2
+echo "Building jar-file..."
+./gradlew bootJar &> /dev/null || { echo "Building jar-file failed!" >&2; exit 1; };
```

Comment:
lets redirect only stdout but stderr leave as is

Reply:
Unfortunately, it looks like separately "> /dev/null"  or  "2> /dev/null" do not solve the problem. The output is still presents in terminal. Seems the only possible way to hide output is use:
1) &>/dev/null
2) >/dev/null 2>&1
3) >/dev/null 2>/dev/null